pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -336,9 +348,26 @@ func (ev *Evaluator) prepareCandidate(ctx context.Context, c Candidate, pod *v1.
 		// Otherwise we should delete the victim.
 		if waitingPod := fh.GetWaitingPod(victim.UID); waitingPod != nil {
 			waitingPod.Reject(pluginName, "preempted")
-		} else if err := util.DeletePod(ctx, cs, victim); err != nil {
-			klog.ErrorS(err, "Preempting pod", "pod", klog.KObj(victim), "preemptor", klog.KObj(pod))
-			return framework.AsStatus(err)
+		} else {
+			if feature.DefaultFeatureGate.Enabled(features.JobPodFailurePolicy) {
+				condition := &v1.PodCondition{
+					Type:    targetedForDisruption,
+					Status:  v1.ConditionTrue,
+					Reason:  "PreemptionByKubeScheduler",
+					Message: "To be preempted by kube-scheduler to accommodate a Pod with higher priority",
```

Comment:
"kube-scheduler: preempted"

Reply:
preempting?

Reply:
Updated as "Kube-scheduler: preempting"

Reply:
Personally I'd use "kube-scheduler" to match the name of the binary, but it doesn't matter that much